gulpfile.js

```diff
@@ -78,7 +78,7 @@ gulp.task('concurrent', () => {
 
 // gulp task to run bower installation
 gulp.task('bower', () => bower({
-  directory: './public/lib'
+  directory: 'public/lib'
```

Comment:
Why change the directory? has the root directory been made static?

Reply:
@andela-abisoye It was a mistake on my path, l would revert it ASAP